### ISSUE TITLE
bCNC autoprobe feature

### ISSUE THREAD

**Issue by atrueresistance** [vlachoudis/bCNC#406]
I am not sure if bCNC has this feature or not.

https://youtu.be/bhZdbgM6S70

I know you can probe the different directions, but not sure about angle correction. I know I would find this super helpful. So does bCNC have this feature?

**Comment by Effer** [comment]
Not exactly the same, the [Camera Alignment](https://github.com/vlachoudis/bCNC/wiki/Probe-Camera-Alignment) allow to adjust a job to fit some reference points. And so it corrects not only the angle, but also the scale of gcode. What is missing is a the z reference. For that you can adjust it manually or using the probe/autolevel feauture.
Anyway it's a nice feature and maybe it can be added as another way for probing.
Thanks for sharing.

**Comment by vlachoudis** [comment]
You can already do it as a macro in one of the programmable buttons

**Comment by atrueresistance** [comment]
Yes, I have the macro buttons configured for zeroing my different sized bits. More or less I was suggesting angle correction feature for a future release.

**Comment by vlachoudis** [comment]
Angle correction , you have already in the Orient frame under probe. Not in such a way but it could be done. All the machinery is there
